cloudbuild.yaml

```diff
@@ -0,0 +1,28 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
```

Comment:
A link to https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md would be useful, because that is where it is specified how this works in Kubernetes.

Reply:
updated the link.